src/locales/tr.json

```diff
@@ -10,14 +10,62 @@
   "footer": {
     "trademark": "Marka",
     "contact": "İletişim ",
-    "policies": "Kullanıcı Sözleşmesi",
+    "policies": "Politikalar",
     "games": "Oyunlar",
     "about": "Hakkında"
   },
-  "resourceSection": {
-    "pollution": "Çevre kirliliği",
-    "climate Change": "Mevsim değişimi",
-    "reduceReuseRecycle": "Azalt, tekrar kullan, geri dönüştür"
+  "login": {
+    "login": "Oturum aç",
+    "email": "Email",
+    "password": "Parola",
+    "forgotPassword": "Şifremi Unuttum?",
+    "new": "juniper yeni misiniz? Şimdi üye Ol",
+    "creat": "Üye Ol",
+    "oryoucan": "Ya da",
+    "withFacebook": "Facebook İle Bağlan",
+    "withGoogle": "Google İle Bağlan"
+  },
+  "passReset": {
+    "emailFormat": "Email biçimi Hatası",
+    "fillField": "Lütfen bu alanı doldurun",
+    "passwordReset": "Şifreyi yenile",
+    "email": "Email",
+    "backToLogin": "Geri Giriş Yap"
+  },
+  "register": {
+    "fullName": "lütfen tam adınızı giriniz",
+    "validEmail": "lütfen geçerli eposta adresini giriniz",
+    "enterAge": "Lütfen yaşını gir",
+    "passwordCharacter": "Şifre en az 6 karakter uzunluğunda olmalı",
+    "enterPassword": "Şifrenizi giriniz Lütfen",
+    "emailFormat": "E-posta biçimi hatası, lütfen geçerli bir e-posta girin",
+    "terms": "Lütfen şartlar ve koşulları kabul edin",
+    "termsTitle": "şartlar ve koşulları",
+    "close": "Kapat",
+    "iAgree": "Kabul ediyorum",
+    "terms1": "1",
+    "terms2": "2",
+    "terms3": "3",
+    "terms4": "4",
+    "terms5": "5",
+    "terms6": "6",
+    "creat": "Yeni hesap oluştur",
+    "name": "Ad Soyad",
+    "age": "Yaş",
+    "password": "Parola",
+    "email": "Email",
+    "iAgreeTo": "Şartları ve koşulları",
+    "termsandCon": "kabul ediyorum",
+    "creatAccount": "Yeni hesap oluştur",
+    "alreadyAMember": "Bir üye misiniz? Şimdi giriş yap",
+    "login": "Giriş ya",
+    "orYouCan": "Ya da",
+    "withFacebook": "Facebook İle Bağlan",
+    "withGoogle": "Google İle Bağlan"
+  },
+  "loginRegister": {
+    "anEmailsent": " By Adresine bir e-posta mesajı gönderildi",
+    "anEmailsentcontain": "şifre sıfırlama talimatları içerir, lütfen gelen kutunuzu kontrol edin ve bu mesajda belirtilen adımları izleyin.."
```

Comment:
This sentence should start wit capitalized letter and you may delete extra dot as well.